src/pages/TodayPage/Components/CreateReminder.tsx

```diff
@@ -0,0 +1,184 @@
+              color='gray'
+              ml='2'
+              _hover={{ color: "red" }}
+              onClick={() => setIsAddTaskOpen(!isAddTaskOpen)}
+            >
+              Görev Ekle
+            </Flex>
+          </Flex>
+          <Centere />
+        </>
+      )}
+      {isAddTaskOpen && (
+        <Flex direction='column' w={isLargerThan800 ? "60%" : "80%"} mt='4'>
+          <Box
+            display='flex'
+            alignItems='center'
+            justifyContent='center'
+            flexDirection='column'
+          >
+            <Flex
+              border='1px'
+              borderColor='gray.300'
+              h='120px'
+              w='100%'
+              borderRadius='xl'
+            >
+              <Box w='200px'>
+                <Input
+                  value={taskName}
+                  onChange={(e) => setTaskName(e.target.value)}
+                  ml='3'
+                  mt='3'
+                  variant='unstyled'
+                  placeholder='Görev ismi'
+                  _placeholder={{ opacity: 1, color: "gray.500" }}
+                />
+                <Input
+                  mt='2'
+                  ml='3'
+                  variant='unstyled'
+                  placeholder='Açıklama'
+                  _placeholder={{ opacity: 1, color: "gray.500" }}
+                />
+                <Flex direction='row' ml='3' mt='2'>
+                  <Flex
+                    border='1px'
+                    borderColor='gray.300'
+                    color='green'
+                    p='2'
+                    mr='2'
+                    borderRadius='md'
+                    w='80px'
+                    h='30px'
+                    alignItems='center'
+                    justifyContent='center'
+                    cursor='pointer'
+                    _hover={{ bg: "gray.200" }}
+                  >
+                    <AiOutlineCalendar />
+                    <Text ml='1'>Bugün</Text>
+                  </Flex>
+                  <Flex
+                    color='blue.400'
+                    flexDirection='row'
+                    border='1px'
+                    borderColor='gray.300'
+                    borderRadius='md'
+                    p='2'
+                    w='80px'
+                    h='30px'
+                    alignItems='center'
+                    justifyContent='center'
+                    cursor='pointer'
+                    _hover={{ bg: "gray.200" }}
+                  >
+                    <AiFillFolderOpen />
+                    <Text color='blackAlpha.700' ml='1'>
+                      Klasör
+                    </Text>
+                  </Flex>
+                </Flex>
+              </Box>
+              <Spacer />
+              <Box
+                display='flex'
+                alignItems='flex-end'
+                justifyContent='flex-end'
+                w='200px'
+                mb='4'
+              >
+                <Text
```

Comment:
in order to dont repeat yourself, create a new array and render those icons.

This component is being repeated for times. 
`<Text fontSize="lg" mr="4" cursor="pointer" _hover={{ bg: 'gray.200' }}>`

You should create icon array and do _**map**_ operation
```
IconList.map(icon=>(
 <Text fontSize="lg" mr="4" cursor="pointer" _hover={{ bg: 'gray.200' }}>
                 {icon}
                </Text>
))
```